packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -43,6 +50,21 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
 
   React.useEffect(
     () => {
+      const pointerTargetRefElement = pointerTargetRef && pointerTargetRef.current
+      const popperHasScrollableParent = getScrollParent(contentRef.current) !== document.body
+
+      const modifiers: PopperJS.Modifiers = _.merge(
+        popperHasScrollableParent && flipForScrollParentModifiers,
```

Comment:
Let's inline them, it's not obvious what `flipForScrollParentModifiers` does

Reply:
done, thanks